service/src/app.impl/systemInfo/app.impl.systemInfo.ts

```diff
@@ -0,0 +1,47 @@
+import { AppResponse } from '../../app.api/app.api.global'
+import * as api from '../../app.api/systemInfo/app.api.systemInfo'
+import { EnvironmentService } from '../../entities/systemInfo/entities.systemInfo'
+import * as Settings from '../../models/setting'
+import * as AuthenticationConfiguration from '../../models/authenticationconfiguration'
+import config from '../../config'
+import AuthenticationConfigurationTransformer from '../../transformers/authenticationconfiguration'
+/**
+ * This factory function creates the implementation of the {@link api.ReadSystemInfo}
+ * application layer interface.
+ */
+export function CreateReadSystemInfo(EnvironmentService: EnvironmentService): api.ReadSystemInfo {
```

Comment:
style nitpick: generally function parameter names should start with a lowercase letter